lib/hocon/impl/simple_config_list.rb

```diff
@@ -149,11 +216,122 @@ def render_value_to_sb(sb, indent_size, at_root, options)
     end
   end
 
-  def new_copy(origin)
-    Hocon::Impl::SimpleConfigList.new(origin, @value)
+  def contains?(o)
```

Comment:
I don't really know if I have an opinion. Ruby does have a habit of giving you 2 different methods that do the same thing =) I'm fine with leaving them in, just thought I'd bring it up